rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.add_rental(self)
+    @person = person
+    person.add_rental(self)
+  end
```

Comment:
- [ ] Given that the `add_rental` method has been updated to fit the requirement, this one here won't work as expected. Kindly refactor your constructor to add itself to the different `rentals` arrays in Person and Book classes.

  ```
    def initialize(date, book, person)
    @date = date
    @book = book
    book << self
    @person = person
    person << self
  end